packages/components/Button/src/ButtonTokens.ios.ts

```diff
@@ -10,75 +10,72 @@ export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = () =>
       width: '100%',
     },
     medium: {
-      padding: globalTokens.size80,
+      paddingHorizontal: globalTokens.size120,
       borderWidth: globalTokens.stroke.width10,
-      iconSize: 16,
+      borderRadius: globalTokens.corner.radius80,
+      minHeight: 40,
+      iconSize: 20,
       focused: {
         borderWidth: 0,
-        padding: globalTokens.size100,
       },
       hasContent: {
         minWidth: 96,
-        paddingHorizontal: globalTokens.size160 - globalTokens.stroke.width10,
         hasIconAfter: {
-          spacingIconContentAfter: globalTokens.size100,
+          spacingIconContentAfter: globalTokens.size80,
         },
         hasIconBefore: {
-          spacingIconContentBefore: globalTokens.size100,
+          spacingIconContentBefore: globalTokens.size80,
         },
         focused: {
-          paddingHorizontal: globalTokens.size160,
+          paddingHorizontal: globalTokens.size120,
```

Comment:
Seems like the focused override for paddingHorizontal can be removed?

Reply:
I wasn't sure you could have an empty or removed "Focused" section and it would behave properly so I thought the safest course was to leave it and it would have no effect.

Reply:
You can remove them safely. Each of the subgroups of tokens are layers which are applied on top of any base or previous layer's values. So if there's a value already defined, it's not required to redefine it to the same value at a later layer. It will have the value that was set by any previous layer, so you only need to define explicit overides when certain layers are applied. The ordering of the layers are usually defined in a <component>.styling.ts file

Reply:
I should also note that you'll basically never get the `focused` state on iOS, given that React Native mobile doesn't really support keyboard focus. So defining tokens here is not really necessary

Reply:
Yeah i've prepped a review to remove this but having trouble running the tests.

Reply:
and pushed